core/src/main/java/org/openstack4j/model/storage/block/EncryptionType.java

```diff
@@ -0,0 +1,45 @@
+package org.openstack4j.model.storage.block;
+
+import org.openstack4j.common.Buildable;
+import org.openstack4j.model.ModelEntity;
+import org.openstack4j.model.storage.block.builder.EncryptionTypeBuilder;
+
+/**
+ * The encryption type defines the characteristics of an encryption.
+ *
+ * @author Jyothi Saroja
+ */
+public interface EncryptionType extends ModelEntity, Buildable<EncryptionTypeBuilder> {
```

Comment:
I am wondering if `VolumeTypeEncryption` would not be more descriptive name here.

Reply:
Yes, this name sounds better, I modified the name now.